app/helpers/application_helper.rb

```diff
@@ -1,2 +1,8 @@
 module ApplicationHelper
+  def flash_class(level)
+    case level.to_sym
+      when :notice then 'success'
+      when :alert then 'danger'
+    end
+  end
```

Comment:
Take a look at: https://www.relishapp.com/rspec/rspec-rails/v/3-5/docs/helper-specs/helper-spec

Basically you want to test if the right string is returned by the helper when you pass in a value to it.